builder/command.go

```diff
@@ -0,0 +1,28 @@
+package builder
+
+type command interface {
+	execute() error
+	undo() error
+}
+
+type stack []command
+
+func newStack() *stack {
+	var s []command
+	return (*stack)(&s)
+}
+
+func (s stack) isEmpty() bool { return len(s) == 0 }
```

Comment:
please change this to multi-line. I had just written it that way in the comments.

Reply:
Okay